packages/firebase_crashlytics/example/android/gradle.properties

```diff
@@ -1 +1,2 @@
 org.gradle.jvmargs=-Xmx1536M
+android.enableR8=true
```

Comment:
This is added automatically everytime `flutter build apk` is called. I'm assuming it is related to changing the `compileSdkVersion`. We would have to add `gradle.properties` to `.gitignore` if we don't want to see this change in every `firebase_crashlytics` PR. For now we can just leave it.